src/main/java/com/bookstore/bookstoreapi/data/controller/BookController.java

```diff
@@ -6,9 +6,7 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
 
 @RestController
 public class BookController {
```

Comment:
Above this line you can add `@RequestMapping(value = "/books")`, and afterwards you can delete the `/books` in your other API mappings below.

Reply:
@airiz04 This is the `Controller` class that your `listCommentsFromBook` need to be situated.